src/components/modals/SupportRequestModal.js

```diff
@@ -0,0 +1,250 @@
+import React, { Component } from 'react';
+import { div, form, input, label, textarea, hh, select, option, section, button } from 'react-hyperscript-helpers';
+import { SupportRequestBaseModal } from '../SupportRequestBaseModal';
+import { Alert } from '../Alert';
+import { Support} from '../../libs/ajax';
+import { Storage } from '../../libs/storage';
+import { Notifications } from '../../libs/utils';
+import CloudUploadIcon from '@material-ui/icons/CloudUpload';
+import HighlightOffIcon from '@material-ui/icons/HighlightOff';
+import Dropzone from 'react-dropzone';
+
+
+export const SupportRequestModal = hh(class SupportRequestModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: Storage.userIsLogged() ? Storage.getCurrentUser().displayName : '',
+      isLogged: Storage.userIsLogged(),
+      type: 'question',
+      subject: '',
+      description: '',
+      attachment: '',
+      email: Storage.userIsLogged() ? Storage.getCurrentUser().email : '',
+      first_name: Storage.userIsLogged() ? Storage.getCurrentUser().displayName.split(' ')[0] : '',
+      height: Storage.userIsLogged() ? '550px': '700px',
+      top: Storage.userIsLogged() ? '400px': '1',
```

Comment:
These settings have an odd effect. If you change window sizes between showing and closing the modal, it remembers the original pixel values and shows up in odd positions along the vertical axis. We should look at how to get it to place itself relative to the screen in a flexible manner.

Reply:
I fixed the effect, but it looks a bit janky to me, we should discuss a cleaner option if possible